console/src/main/java/org/matrix/console/services/EventStreamService.java

```diff
@@ -640,12 +635,19 @@ private Notification buildNotification() {
                 Intent.FLAG_ACTIVITY_SINGLE_TOP);
 
         PendingIntent pi = PendingIntent.getActivity(this, 0, i, 0);
-
-        notification.setLatestEventInfo(this, getString(R.string.app_name),
```

Comment:
`setLatestEventInfo` was removed, so I needed to build a `Notification` using the `NotificationCompat.Builder` mechanism